components/fxa-client/src/oauth.rs

```diff
@@ -78,6 +89,10 @@ impl FirefoxAccount {
         self.oauth_flow(url, scopes, true)
     }
 
+    /// Initiate an OAuth login flow and return a URL that should be navigated to.
+    ///
+    /// * `scopes` - Space-separated list of requested scopes.
+    /// * `wants_keys` - Retrieve scoped keys associated with scopes supporting it.
```

Comment:
Not for this PR, but I feel like we should be able to get rid of this `wants_keys` parameter somehow; whether or not you want keys is implicit in the scopes that you're requesting.